app/src/main/java/com/akshay/newsapp/NewsApp.kt

```diff
@@ -1,22 +1,20 @@
 package com.akshay.newsapp
 
 import android.app.Application
-import com.akshay.newsapp.di.base.AppInjector
-import dagger.android.AndroidInjector
-import dagger.android.DispatchingAndroidInjector
-import dagger.android.HasAndroidInjector
-import javax.inject.Inject
+import com.akshay.newsapp.koin.appModule
+import org.koin.android.ext.koin.androidContext
+import org.koin.core.context.startKoin
 
-class NewsApp : Application(), HasAndroidInjector {
-
-    @Inject
-    lateinit var androidInjector: DispatchingAndroidInjector<Any>
+class NewsApp : Application() {
 
     override fun onCreate() {
         super.onCreate()
-        // Init DI magic ✨
-        AppInjector.init(this)
+        // start Koin!
+        startKoin {
```

Comment:
Is it possible to move this logic of `startKoin` into `AppInjector.init` so we have a layer of abstraction?

Reply:
I am not sure about this. I have always added the startKoin logic here. Also, it would be best to remove all the dagger related dependencies from the project as we are switching over to Koin. 
Let me know your thoughts on this.

Reply:
`AppInjector` class is not part of Dagger dependency -> it's the abstraction over Dagger

Reply:
Okay. Got it. Let me see if this can be done.

Reply:
Thanks. It would be great to have else the last option would be creating a base application class which I don't prefer but let's see 🤷‍♂

Reply:
Done 👍